### DISCUSSION TITLE
Transform erroneous fields to null?

### DISCUSSION THREAD

**Discussion by kainbryanjones** [colinhacks/zod#3140] [Q&A] [answered]
Say I have a large object with various fields. I want to basically parse it so if a partciular field fails to parse its value just becomes null, thus the rest of the object can still parse and one failure doesn't lead to the whole parse failing.

**Comment by JacobWeisenburger** [comment, marked as answer]
perhaps this would be helpful:
https://github.com/JacobWeisenburger/zod_utilz#partialsafeparse
